stats/stats.go

```diff
@@ -87,7 +88,7 @@ func getStatsFromRepo(repo, tempPath, provider string, l *log.Logger) (repositor
 
 	tDownloader := downloader.(repository.Downloader)
 
-	path, err = tDownloader.Download(repo, tempPath, l)
+	path, err = tDownloader.Download(repo.URL, tempPath, repo.DefaultBranch, l)
```

Comment:
why not pass repo an access `repo.URL` and `repo.DefaultBranch` ?

Reply:
totally agree with that :)